lib/edition_filter.rb

```diff
@@ -73,6 +73,12 @@ def filtered_scope(scope)
         end
       when :organisation
         memo.merge(TagsRevision.tagged_organisations_include(value))
+      when :political
+        if value == "include_political"
+          memo.political
+        else
+          memo.political(false)
```

Comment:
Can probably simplify this down to `memo.political(value == "include_political")`